src/nginxconfig/i18n/en/templates/global_sections/tools.js

```diff
@@ -37,9 +37,17 @@ export default {
     resetGlobalConfig: 'Reset global config',
     resetAllDomains: 'Reset all domains',
     removeAllDomains: 'Remove all domains',
+    removeAllDomainsConfig: 'Reset all domain configs',
```

Comment:
```suggestion
    resetAllDomainsConfig: 'Reset all domain configs',
```
use `reset` to be clearer